.travis.yml

```diff
@@ -7,28 +7,25 @@ addons:
       - python3
       - python3-yaml
 install: true
-script:
-- ./gradlew --no-daemon check
-after_success:
-- chmod +x ./.travis/update_checkstyle_thresholds && ./.travis/update_checkstyle_thresholds
+after_success: ./.travis/update_checkstyle_thresholds
 before_deploy:
 ## Run the travis setup, will download install4j and configure it
-- chmod +x ./.travis/setup && ./.travis/setup
+- ./.travis/setup
 ## Add the travis build number to the game engine version
 - GAME_CONFIG=game_engine.properties
 - ENGINE_VERSION=$(grep "engine_version" $GAME_CONFIG | sed 's/.*= *//g')
 - export TAGGED_VERSION="$ENGINE_VERSION.$TRAVIS_BUILD_NUMBER"
 - sed -i "s/engine_version.*/engine_version = $TAGGED_VERSION/" $GAME_CONFIG
 ## Run the gradle release build process - creates the installers
-- ./gradlew --no-daemon release
```

Comment:
`--no-daemon` was added to the `script` phase (`check` task) in 719397d as an optimization:

```
Add no-daemon flag to gradle run, since we are doing a one-off run in a container, the daemon does not help
```

And then added to the `before_deploy` phase (`release` task) in 9fcf6f7.

Since you removed the `script` phase and are relying on the default Travis Gradle build command of `./gradlew check` (i.e. using a daemon), I think it makes sense to remove `--no-daemon` from the `release` task because a daemon is already running at this point.